modules/server/src/main/ts/bedrock/compiler/Webpack.ts

```diff
@@ -54,9 +54,13 @@ const webpackSharedRules = webpackRemap.concat([
     enforce: 'pre'
   },
   {
-    test: /\.(html|htm|css|bower|hex|rtf|xml|yml)$/,
+    test: /\.(html|htm|css|bower|hex|rtf|xml|yml)$/i,
     type: 'asset/source'
   },
+  {
+    test: /\.(jpe?g|png|gif|svg)$/i,
```

Comment:
```suggestion
    test: /\.(jpe?g|png|gif|svg|apng|avif|webp|bmp|tiff)$/i,
```
This just adds in more image formats to save us running into issues later (it should be everything we include for TinyMCE).

Reply:
Actually there are a few more weird variants like `jpe`, `jfi`, `jif`, and `jfif` if you wanted to include them, but I'm less concerned about them 🤷

Reply:
I'm not sure we need any of them, but sure :)